tests/unittests/OnnxExporterTest.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/Exporter/ONNXModelWriter.h"
+#include "glow/Graph/Graph.h"
+#include "glow/Graph/Nodes.h"
+#include "glow/Graph/PlaceholderBindings.h"
+#include "glow/Importer/ONNXModelLoader.h"
+#include "gtest/gtest.h"
+
+#include "llvm/Support/FileSystem.h"
+
+#ifndef GLOW_DATA_PATH
+#define GLOW_DATA_PATH
+#endif
+
+using namespace glow;
+
+namespace {
+void testLoadAndSaveONNXModel(const std::string &name) {
+  ExecutionEngine EE{};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  llvm::Error err = llvm::Error::success();
+  ONNXModelLoader onnxLD(name, {}, {}, *F, &err);
+
+  if (!err) {
```

Comment:
Use early exit instead? And maybe show a message that an ONNX model could not be loaded.